Sources/TestHelper/NavigationPlotter.swift

```diff
@@ -160,14 +160,14 @@ public class NavigationPlotter: UIView {
     
     func updateCoordinateBounds() {
         navigationMapView = NavigationMapView(frame: bounds)
-        let padding = UIEdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
+//        let padding = UIEdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
         
         coordinateBounds = allBoundingCoordinates.bounds
         
-        if let coordinateBounds = coordinateBounds {
+//        if let coordinateBounds = coordinateBounds {
             // FIXME: As of Mapbox Maps `v10.0.0-beta.17` such method is no longer present.
             // navigationMapView?.mapView.cameraManager.transitionCoordinateBounds(to: coordinateBounds, edgePadding: padding, completion: nil)
```

Comment:
If I’m not mistaken, it should be possible to fit a camera to these bounds using `MapboxMap.camera(for:padding:bearing:pitch:)`, then set the camera.

Reply:
Even though I can verify that it will work at this point, I will update the code with the suggested replacement.